### PR TITLE
Twitter Streams demo

### PR DESCRIPTION
**Pull request by gAmUssA**
- updated to 5.0.1
- uses ZK, 2 AK, SR, KSQL, C3, custom connect image with `kafka-connect-twitter`

### CODE REVIEW

File: twitter-streams/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  kafka1:
+    image: confluentinc/cp-enterprise-kafka:5.0.1
+    hostname: kafka1
+    depends_on:
+      - zookeeper
+    # persisting data in local fs
+    volumes:
+      - $PWD/data/kafka1/data:/var/lib/kafka/data
+    ports:
+      - "9091:9091"
```
Comment (rmoff) [inline twitter-streams/docker-compose.yml]: nit: no point exposing this as it's the internal listener

File: twitter-streams/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+  # Begin kafka2 section
+  kafka2:
+    image: confluentinc/cp-enterprise-kafka:5.0.1
+    hostname: kafka2
+    depends_on:
+      - zookeeper
+    volumes:
+      - $PWD/data/kafka2/data:/var/lib/kafka/data
+    ports:
+      - "9092:9092"
```
Comment (rmoff) [inline twitter-streams/docker-compose.yml]: nit: no point exposing this as it's the internal listener

File: twitter-streams/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+      - "29092:29092"
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka2:9092,PLAINTEXT_HOST://localhost:29092
```
Comment (rmoff) [inline twitter-streams/docker-compose.yml]: FWIW I put this in all my Docker Compose files to help people copy & pasting and debugging their own setups: https://github.com/confluentinc/demo-scene/blob/master/mysql-debezium-ksql-elasticsearch/docker-compose/docker-compose.yml#L15-L17 /

Comment (gAmUssA) [reply twitter-streams/docker-compose.yml]: 👍

**Pull request merged**